pages/ksphere/konvoy/latest/install/node-pools/index.md

```diff
@@ -10,7 +10,7 @@ enterprise: false
 In this section, you will learn how to configure multiple node pools for a Konvoy cluster.
 Node pools allow the cluster administrator to use different configurations for different sets of worker nodes in a heterogeneous environment.
 
-<p class="message--important"><strong>IMPORTANT: </strong> Do not change the configuration of a deployed node pool. Changing the configuration of a deployed node pool, and running  `konvoy provision` or `konvoy up`, destroys and re-creates the machines, disrupting your workloads on these machines. To change the configuration of a node pool, see <a href="../../operations/managing-nodes/index.md#changing-nodes-in-an-aws--azure-cluster">Changing Nodes in an AWS / Azure Cluster</a>.</p>
+<p class="message--important"><strong>IMPORTANT: </strong> Do not change the configuration of a deployed node pool. Changing the configuration of a deployed node pool, and running  `konvoy provision` or `konvoy up`, destroys and re-creates the machines, disrupting your workloads on these machines. To change the configuration of a node pool, see <a href="/ksphere/konvoy/latest/operations/managing-nodes/#changing-nodes-in-an-aws-azure-cluster">Changing Nodes in an AWS / Azure Cluster</a>.</p>
```

Comment:
```suggestion
<p class="message--important"><strong>IMPORTANT: </strong> Do not change the configuration of a deployed node pool. Changing the configuration of a deployed node pool, and running  `konvoy provision` or `konvoy up`, destroys and re-creates the machines, disrupting your workloads on these machines. To change the configuration of a node pool, see <a href="../../operations/managing-nodes/#changing-nodes-in-an-aws-azure-cluster">Changing Nodes in an AWS / Azure Cluster</a>.</p>
```

Will using `../../` still work? We have `../../operations` on other places in the doc:
https://docs.d2iq.com/ksphere/konvoy/latest/install/install-airgapped/#checking-the-files-installed
And click on
>admin.conf - is a kubeconfig file, which contains credentials to connect to the kube-apiserver of your cluster through kubectl.

Reply:
Yes, I just now verified using `../../` also works. I'm not sure which method is more "correct" so let's go with your suggestion since it matches other links.